core/scripts/keystone/main.go

```diff
@@ -15,13 +15,16 @@ type command interface {
 
 func main() {
 	commands := []command{
-		src.NewDeployContractsCommand(),
-		src.NewDeployJobSpecsCommand(),
-		src.NewGenerateCribClusterOverridesCommand(),
-		src.NewDeleteJobsCommand(),
+		src.NewProvisionKeystoneCommand(),
+		src.NewFundTransmittersCommand(),
+		src.NewProvisionOCR3CapabilityCommand(),
+		src.NewDeployKeystoneWorkflowsCommand(),
+		src.NewDeployForwarderCommand(),
+		src.NewPreprovisionCribCommand(),
+		src.NewPostprovisionCribCommand(),
+		src.NewProvisionCapabilitesRegistryCommand(),
+		src.NewProvisionStreamsTriggerCommand(),
 		src.NewDeployAndInitializeCapabilitiesRegistryCommand(),
```

Comment:
@cedric-cordenier Can we remove this? I kept it around because I wasnt sure if the team needs this. `NewProvisionCapabilitesRegistryCommand` handles provisioning CR for crib

Reply:
Yes we can remove this if `NewProvisionCapabilitesRegistryCommand` handles it.